frontend/src/store/store.js

```diff
@@ -0,0 +1,20 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import loginReducer from './login/reducers';
+// import registerReducer from './register/reducer';
```

Comment:
comments here and a few other places like line 13. I'm okay with documentation just not commented out code